src/components/SearchMap.js

```diff
@@ -157,7 +83,9 @@ const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites, navigate, s
 
                     </MapView>
 
-                    {renderSelectedSitePreview()}
+                    <SiteCarousel
+                        navigate={navigate}
+                    />
```

Comment:
Much cleaner call to the `SiteCarousel` now. Much smaller components overall.

Reply:
Awesome

Reply:
Awesome